task1/task1.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Task 1</title>
+</head>
+
+<body>
+    <table border=1 cellspacing=0 cellpadding=5px>
+        <thead>
+            <tr height=100px>
+                <th colspan="3">
+                    <font size="6">Markup of “Basic HTML” task</font>
+                </th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr valign=top>
+                <td width=200px>
+                    <table>
```

Comment:
Do you really need the table to create a list?

Reply:
Yes, ul an ol have bullets and numbers showing up next to items and you cannot hide them without CSS, therefore I used a table for the list to appear like in mockup

Reply:
And `type=none` with ul list makes bullets disappear but leaves a padding.